docs/user/alerting/rule-types/index-threshold.asciidoc

```diff
@@ -1,100 +1,108 @@
+In this example, you will use the {kib} <<add-sample-data,sample weblog data set>> to set up and tune the conditions on an index threshold rule. For this example, you want to detect when any of the top four sites serve more than 420,000 bytes over a 24 hour period.
 
 .  Open the main menu, then click *{stack-manage-app} > {rules-ui}*.
 
-.  Create a new rule that is checked every four hours and triggers actions when the rule status changes.
+.  Create a new rule.
+
+.. Provide a rule name and select the **Index threshold** rule type.
 +
 [role="screenshot"]
 image::user/alerting/images/rule-types-index-threshold-select.png[Choosing an index threshold rule type]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
 
-.  Select the **Index threshold** rule type.
-
-. Click *Index*, and set *Indices to query* to *kibana_sample_data_logs*.
+.. Select an index. Click *Index*, and set *Indices to query* to `kibana_sample_data_logs`. Set the *Time field* to `@timestamp`.
 +
 [role="screenshot"]
 image::user/alerting/images/rule-types-index-threshold-example-index.png[Choosing an index]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
 
-. Set the *Time field* to *@timestamp*.
-+
-[role="screenshot"]
-image::user/alerting/images/rule-types-index-threshold-example-timefield.png[Choosing a time field]
-
-. To detect the number of bytes served during the time window, click *When* and select `sum` as the aggregation, and bytes as the field to aggregate.
+.. To detect the number of bytes served during the time window, click *When* and select `sum` as the aggregation, and `bytes` as the field to aggregate.
 +
 [role="screenshot"]
 image::user/alerting/images/rule-types-index-threshold-example-aggregation.png[Choosing the aggregation]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
 
-. To detect the four sites that have the most traffic, click *Over* and select `top`, enter `4`, and select `host.keyword` as the field.
+.. To detect the four sites that have the most traffic, click *Over* and select `top`, enter `4`, and select `host.keyword` as the field.
 +
 [role="screenshot"]
 image::user/alerting/images/rule-types-index-threshold-example-grouping.png[Choosing the groups]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
 
-. To trigger the rule when any of the top four sites exceeds 420,000 bytes over a 24 hour period, select `is above` and enter `420000`.
+.. Define the condition.
 +
 [role="screenshot"]
 image::user/alerting/images/rule-types-index-threshold-example-threshold.png[Setting the threshold]
 
-. Finally, click *For the last*, enter `24` and select `hours` to complete the rule configuration.
+... To trigger the rule when any of the top four sites exceeds 420,000 bytes over a 24 hour period, select `is above` and enter `420000`. Then click *For the last*, enter `24`, and select `hours`.
```

Comment:
nit: the associated screenshot shows a threshold of 4200. not sure how important it is that they match.

Reply:
Good catch!  I missed a zero!

Reply:
I've now automated that screenshot, so this issue is now fixed.